SignalMessaging/utils/DisplayableText.swift

```diff
@@ -215,6 +215,17 @@ extension String {
         return text.ows_stripped()
     }
 
+    @objc
+    public class func filterNotificationText(_ text: String?) -> String? {
+        guard let text = self.filterText(text) else {
+            return nil
+        }
+
+        // Notifications strip anything that looks lik a printf formatting character,
```

Comment:
s/lik/like/

Reply:
s/printf formatting character/string formatting specifier/

Reply:
typo fixed - thanks. 

To be consistent with the iOS docs, I'm going to continue to refer to the affected chars as "printf formatting" chars.